plugins/main/public/components/overview/it-hygiene/users/inventory.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {
+  ITHygieneUsersInventoryUsers,
+  ITHygieneUsersInventoryGroups,
+} from './inventories';
+import { ITHygieneInventoryTabLayout } from '../common/components/inventory';
+
+const tabs = [
+  {
+    id: 'users',
+    name: 'Users',
+    component: ITHygieneUsersInventoryUsers,
+  },
+  {
+    id: 'groups',
+    name: 'Groups',
+    component: ITHygieneUsersInventoryGroups,
+  },
+];
+
+export const ITHygieneUsersInventory = () => {
+  return <ITHygieneInventoryTabLayout tabs={tabs} />;
+};
```

Comment:
Great work so far

One minor sugesstion: 
would not be better to create a singe reusable component  that may be populated with props like: `columns, filters, datasource` - it would cut down on duplicated code between this two modules: users, groups

so 
instead of having `users/inventory.tsx` and `groups/inventory.tsx`, you would have `genericInventory.tsx`